src/ducks/resource.js

```diff
@@ -1,12 +1,46 @@
 import apiClient from 'panoptes-client/lib/api-client';
 
 // Action Types
-export const FETCH_RESOURCE = 'FETCH_RESOURCE';
-export const FETCH_RESOURCE_SUCCESS = 'FETCH_RESOURCE_SUCCESS';
-export const FETCH_RESOURCE_ERROR = 'FETCH_RESOURCE_ERROR';
+export const FETCH_TRANSLATIONS = 'FETCH_TRANSLATIONS';
+export const FETCH_TRANSLATIONS_SUCCESS = 'FETCH_TRANSLATIONS_SUCCESS';
+export const FETCH_TRANSLATIONS_ERROR = 'FETCH_TRANSLATIONS_ERROR';
+export const CREATE_TRANSLATION = 'CREATE_TRANSLATION';
 export const CREATE_TRANSLATION_SUCCESS = 'CREATE_TRANSLATION_SUCCESS';
+export const SELECT_TRANSLATION = 'SELECT_TRANSLATION';
```

Comment:
Create should only be called when the translation doesn't exist. There's also a `selectTranslation` action which returns an existing translation, or creates one if an existing one isn't found.

Create and select both invoke the same reducer, but they do slightly different things.